lib/api/statistics.js

```diff
@@ -317,7 +323,7 @@ module.exports = function (app) {
 			p.osBrand = osBrand(p.os);
 			this.ips.push(p.ip);
 
-			switch (parseInt(p.state, 10)) {
+			switch (stateMapping[p.state]) {
```

Comment:
Does it keep compatibility with the Core 2.x?
In case it does not, it could distinguish it between string and number and apply the right value.

Reply:
Good catch. I totally forgot about compatibility with the Core 2.x. I don't see how it happened, given how much I argued for backwards compatibility in the Core 3.0 STM 🙂